lib/transactions/delegate.js

```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating delegates in Lisk Blockchain
```

Comment:
Delegates module provides functions to create delegates for the Lisk Blockchain.